syntax = "proto3";

service GrpcAuthors {
    rpc GetAuthor(GetAuthorRequest) returns (AuthorGrpc) {}
}

message GetAuthorRequest {
    int32 id = 1;
}

message AuthorGrpc {
    int32 id = 1;
    string jp_name = 2;
    string romaji_name = 3;
    string twitter = 4;
}
